backend/src/packages/users/user.controller.ts

```diff
@@ -166,6 +177,29 @@ class UserController extends Controller {
     };
   }
 
+  /**
+   * @swagger
+   * /articles/activity:
+   *    get:
+   *      summary: Get user activity statistic
+   *      description: Get user activity statistic
+   *      responses:
+   *        200:
+   *          description: Successful operation
+   *          content:
+   *            application/json:
+   *              schema:
+   *                $ref: '#/components/schemas/Article'
```

Comment:
I guess the response schema should not be `Article`, but `shared/src/packages/users/libs/types/user-activity-response-dto.type.ts`